mirdata/utils.py

```diff
@@ -217,8 +221,16 @@ def download_from_remote(remote, data_home=None, clobber=False):
         with DownloadProgressBar(unit='B', unit_scale=True,
                                  miniters=1,
                                  desc=remote.url.split('/')[-1]) as t:
-            urlretrieve(
-                remote.url, filename=download_path, reporthook=t.update_to)
+            try:
+                download_large_file(remote.url, download_path, t.update_to)
+            except requests.exceptions.HTTPError:
+                error_msg = 'mirdata failed to download {}! \
+                             Please try again in a few minutes. \
+                             If this error persists, please raise an issue at \
+                             https://github.com/mir-dataset-loaders/mirdata, \
+                             and tag it with "broken-link". \
+                             '.format(remote.filename)
+                raise IOError(error_msg)
```

Comment:
Maybe create a custom error. Maybe use one of these instead https://docs.python.org/3/library/exceptions.html#ConnectionError

Reply:
Why not just re-raise the `HTTPError` with the appropriate message? That's what is happening here